### PR TITLE
Git ignore .import folder

### LINKED ISSUE #92: Add `.import` folder to .gitignore
The .import folder inside the custom mod you all created should not be checked in to git as it really slows down bloats the repository and is not required for the mod to function.

Remove the folder and add it to the .gitignore so that future revisions don't add anymore bloat, sadly unless we do some surgery it will always stay in the history.

### PR DESCRIPTION
**Pull request by Adrianmeda**
⛓ **Description:**

- Close #92 (add .import to .gitignore)

Deleted and added .import folder to .gitignore as requested of @RuthgerD 

🛠 **Type of change:**
- [x] .import folder deleted
- [x] .import added to .gitignore

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -72,3 +72,6 @@ google-services.json
 
 # MSC
 .DS_Store
+
+# ignore .import folder
+*.import
```
Comment (RuthgerD) [inline .gitignore:77]: This also ignores files, which ironically enough is not what we want :)

either specify the path or use a glob pattern for directories: `**/.import/`

File: arduino/smartcar/smartcar.ino [hunk 2]
```diff
@@ -118,9 +118,24 @@ void takeInput(String input) {
...
+     car.setSpeed(-10); // incase the car was going to fast and didnt manage to stop in time before hitting the object
+     delay(900);
+     car.setSpeed(15);
+  }
+} // method finished and can continue with the manual input
+
 
 void loop() {
- 
+ obstacleAvoid ();
```
Comment (RuthgerD) [inline arduino/smartcar/smartcar.ino:138]: this is an unrelated change

**Review by RuthgerD — CHANGES_REQUESTED** [review]

File: arduino/smartcar/smartcar.ino (hunk 2 above)
Comment (Adrianmeda) [reply arduino/smartcar/smartcar.ino:138]: Yes wasnt meant to be added.

**Review by RuthgerD — APPROVED** [review]
